pyproject.toml

```diff
@@ -28,4 +28,4 @@ use_parentheses = true
 skip_glob = ["build/*", "stubs/*"]
 # Don't split "import" and "from".
 force_sort_within_sections = true
-known_third_party = ["benchmark_dataset", "dataclasses", "numpy", "pytest", "recommonmark", "setuptools", "torch", "torch_pg", "torchtext", "torchvision"]
+known_third_party = ["benchmark_dataset", "dataclasses", "helpers", "numpy", "pytest", "recommonmark", "setuptools", "torch", "torch_pg", "torchtext", "torchvision"]
```

Comment:
where is helpers from? I didn't see it in this PR.

Reply:
oh, I've absolutely no idea where that came from VSCode shenanigan I suppose, removing that

Reply:
the git commit hooks are the ones creating this file, not how that works to be honest